Docker Compose Bug

How to reproduce the behaviour
---------
<!-- Before submitting an issue, make sure to check the docs and closed issues and FAQ to see if any of the solutions work for you. https://github.com/chakki-works/doccano/wiki/Frequently-Asked-Questions -->

`docker-compose up`

<!-- Include a code example or the steps that led to the problem. Please try to be as specific as possible. -->

`django_1    |     In file included from src/buffer.cpp:12:
django_1    |     src/pyodbc.h:56:10: fatal error: sql.h: No such file or directory
django_1    |      #include <sql.h>`

Your Environment
---------
<!-- Include details of your environment.-->
*   Operating System: Mac Mojave 10.14
*   Python Version Used: 3.6
*   When you install doccano: 10/7/19
*   How did you install doccano (Heroku button etc): locally - clone the repo

--- a/tools/dev-django.sh
+++ b/tools/dev-django.sh
@@ -14,6 +14,7 @@ if [[ ! -f "${venv}/bin/python" ]]; then
 fi
 
 echo "Installing dependencies"
+apt-get update && apt-get install -y g++ unixodbc-dev # pyodbc build dependencies
 "${venv}/bin/pip" install -r "${root}/requirements.txt"
 
 echo "Initializing database"

